### ISSUE TITLE
Flexible naming for forcing variables

### ISSUE THREAD

**Issue by ShervanGharari** [CH-Earth/summa#557]
#### Flexible naming for forcing variables

- Currently, the forcing variables in input file(s) should be changed to fixed names required by SUMMA. This adds to the model-specific tasks for preparing the SUMMA forcing.
- Will the proposed feature be backward compatible? depending on the design it can be backward compatible. The forcing variables can be set internally to the names that are currently required. If otherwise provided by users, those names will be changed, otherwise, they remain as before which allows earlier setup to be simulated without changing anything in the setting file or decision file.
- Will the proposed feature change the science results of SUMMA? No, but reduces the forcing preparation time and maybe duplication of forcing for summa.

**Comment by andywood** [comment]
How will the forcing file variables be specified?  An additional input file or modifications one of the existing input files?  I'd probably do it by adding an additional but optional one-liner parsable string in the main control file.  If not included, default behavior is maintained.

**Comment by ShervanGharari** [comment]
Yes, it can be a separate file, but I would suggest for brevity to have it in the `fileManager.txt` because it is related to the forcing files at the end of the day, we can have some more optional info about the variable names or dt in that file.
```
dt              3600 # if not provided use nc file time stamp from the forcing file #556
SWR         dswr # name of variables in the NC file for downward short-wave radiation
.
.
.
```
The names of the variables can be initialized as a global parameter and if they are mentioned in the `fileManager.txt`, they can be renamed and passed to SUMMA for reading. This makes the code backward compatible also.
As you mentioned, a similar approach can be done in a separate file called `forcingFileMeta.txt` for example, and be added to the file manager as an optional line. A bit more work but cleaner than having it in the `fileManager.txt`.

**Comment by andywood** [comment]
The simplest approach would be one new optional, parseable line in
filemanager, eg:
forcingVarNames         pptrate:prec,airtemp:tair,windspd:windspeed

No new input file with only a small amt of info to explain and maintain in
a template setup.
